.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = tab
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.json,*.yaml,*.yml]
```

Comment:
editorconfig now requires that we add brackets when we have comma. Not sure why - it's erroring out in PHPStorm at the very least.

```suggestion
[{*.json,*.yaml,*.yml}]
```
Or `[*.{json,yaml,yml}]`